rt/frontend/simple/src/main/java/org/apache/cxf/endpoint/dynamic/TypeClassInitializer.java

```diff
@@ -50,6 +50,15 @@
     boolean isFault;
     Bus bus;
 
+    @Deprecated // CXF-8658
+    public TypeClassInitializer(ServiceInfo serviceInfo,
+            S2JJAXBModel model,
+            boolean allowWr) {
+        super(serviceInfo);
```

Comment:
```suggestion
        this(BusFactory.getDefaultBus(), serviceInfo, model, allowWr);
```
I think this is the best we can do there ... :-\